commit c6df8f14dace302ded7cb4ab8dee817d4e128687
Author: Georges de Massol <jojo2massol@gmail.com>
Date:   Wed Dec 14 18:42:19 2022 +0100

    patch for html server

diff --git a/examples/websocket-server/main.c b/examples/websocket-server/main.c
index cacd2662..02ad5e51 100644
--- a/examples/websocket-server/main.c
+++ b/examples/websocket-server/main.c
@@ -5,8 +5,8 @@
 
 #include "mongoose.h"
 
-static const char *s_listen_on = "ws://localhost:8000";
-static const char *s_web_root = ".";
+static const char *s_listen_on = "ws://0.0.0.0:8000";
+static const char *s_web_root = "/var/www/html";
 
 // This RESTful server implements the following endpoints:
 //   /websocket - upgrade to Websocket, and implement websocket echo server
